crates/indexer/src/http/store_config.rs

```diff
@@ -24,11 +24,23 @@ pub struct Address {
     pub auctionHouse: String,
```

Comment:
```suggestion
    pub auction_house: String,
```

Add `#[serde(rename_all = "camelCase")]` in between line 21 and 22.